lib/browser/api/browser-view.js

```diff
@@ -1,9 +1,59 @@
 'use strict'
 
+const electron = require('electron')
+const {ipcMain} = electron
 const {EventEmitter} = require('events')
 const {BrowserView} = process.atomBinding('browser_view')
+const v8Util = process.atomBinding('v8_util')
 
 Object.setPrototypeOf(BrowserView.prototype, EventEmitter.prototype)
+BrowserView.prototype._init = function () {
+  // Make new windows requested by links behave like "window.open"
+  this.webContents.on('-new-window', (event, url, frameName, disposition,
```

Comment:
Maybe we could just do this in `lib/browser/api/web-contents.js` to share the impl between BrowserWindow/BrowserView? cc @zcbenz

Reply:
@Anrock Feel free to just make new commits. We can squash it all when merging.